seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,269 @@
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'The following families are included in the callset but are missing some family members: 1 (NO_MATCHED_SAMPLE_IND_ID).')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        mock_get_es_samples.assert_called_with(ELASTICSEARCH_INDEX)
+
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.logger')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_elasticsearch_index_samples')
+    def test_command_error_missing_families(self, mock_get_es_samples, mock_logger):
+        mock_get_es_samples.return_value = SAMPLE_IDS, INDEX_METADATA
+
+        # Create a new SavedVariant record which belongs to a family missed
+        family = Family.objects.get(pk=3)
+        saved_variant = SavedVariant.objects.create(
+            xpos_start = 21005543353,
+            xpos_end = 21005543356,
+            ref = "GAGA",
+            alt = "G",
+            variant_id = "21-5543353-GAGA-G",
+            family = family
+        )
+        saved_variant.save()
+
+        with self.assertRaises(CommandError) as ce:
+            call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'The following families have saved variants but are missing from the callset: 3.')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        mock_get_es_samples.assert_called_with(ELASTICSEARCH_INDEX)
+
+    @mock.patch.object(__builtin__, 'raw_input')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.logger')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_elasticsearch_index_samples')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_es_variants_for_variant_tuples')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_single_es_variant')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.LiftOver')
+    def test_command_other_exceptions(self, mock_liftover, mock_single_es_variants, mock_get_es_variants, mock_get_es_samples, mock_logger, mock_input):
+        mock_get_es_samples.return_value = SAMPLE_IDS, INDEX_METADATA
+
+        # Update a SavedVariant record to be a lifted variant
+        saved_variant = SavedVariant.objects.get(
+            variant_id = "21-3343353-GAGA-G",
+        )
+        saved_variant.saved_variant_json['genomeVersion'] = GENOME_VERSION_GRCh38
+        saved_variant.save()
```

Comment:
Again, please create fixture data. If you find that adding fixture data causes unrelated tests to fail, you might want to try using project 3 instead of project 1, as it is used in far fewer other tests

Reply:
It requires too much work to create new saved_variants for another project. Therefore, I keep using project 1. I changed the save_variant to have 'genomeVersion' 38 directly in the fixture data. It seems work.